free-1.md

```diff
@@ -57,12 +95,24 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think time would be split more towards systems at the beginning, when making an
+MVP. But once we progress past single command=single input, I have been extremely
+conflicted about how to deal with any kind of global state or conditionals, which
+can be very important in some games.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is a project that should be relatively easy to get something working, but new
```

Comment:
There may be a lot of work that goes into learning the Twitch API and designing the language around that, unless you are already very familiar with it. And also, because it seems like Pokemon is the only major Twitch Play that's happened so far, you might need to determine how your DSL will interact with different kinds of games.